### PR TITLE
all: go-llvm that defaults to llvm-12

### PR DESCRIPTION
**Pull request by kenbell**
`go install` now uses LLVM 12 by default, `go install -tags llvm11` will use LLVM 11.

### CODE REVIEW

**Review by aykevl — APPROVED** [review]

**Comment by kenbell** [comment]
Build is breaking because I didn't test `go install -tags llvm12` when I updated `go-llvm`, this go-llvm PR (https://github.com/tinygo-org/go-llvm/pull/28) needs to be merged, then i can update this PR.

**Review by aykevl — APPROVED** [review]
LGTM, waiting for CI to give the green light.

**Comment by deadprogram** [comment]
All green, so now merging. Thank you @kenbell

**Comment by aykevl** [comment]
Actually this is missing something: cgo/libclang_config*.go must also be updated. I missed this in my review.
But it's a bit complicated because darwin still defaults to LLVM 11.

**Pull request merged**
